CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
   - fixed help links in configuration overlay
 - Notification
   - fixed help link
+- Unhelpful scroll
+  - fixed auto focus issue which was resulted in unhelpful scroll in Partner Network and Application Request page
```

Comment:
looks good in general, could you please fix the sentence ...which ~was~ resulted in...